docs/content/users/extend/additional-hostnames.md

```diff
@@ -6,13 +6,15 @@ Add additional hostnames to a project in its `.ddev/config.yaml`:
 name: mysite
 
 additional_hostnames:
-- "extraname"
-- "fr.mysite"
-- "es.mysite"
-- "it.mysite"
-- "*.lotsofnames"
+  - "extraname"
+  - "fr.mysite"
+  - "es.mysite"
+  - "it.mysite"
+  - "*.lotsofnames"
 ```
 
+Hostnames can be added with the console command `ddev config --additional-hostnames comma,separated,hostnames`
+
 This configuration would result in working hostnames of `mysite.ddev.site`, `extraname.ddev.site`, `fr.mysite.ddev.site`, `es.mysite.ddev.site`, and `it.mysite.ddev.site`—with full HTTP and HTTPS URLs for each.
 
```

Comment:
I’d re-add this here, and edit to clarify this is another way of doing the same thing as the preceding example and use the same values to illustrate the difference.

(I four-space-indented the snippet since I can’t add a fenced code block within a suggestion.)

```suggestion
This configuration would result in working hostnames of `mysite.ddev.site`, `extraname.ddev.site`, `fr.mysite.ddev.site`, `es.mysite.ddev.site`, and `it.mysite.ddev.site`—with full HTTP and HTTPS URLs for each.

You could accomplish the same thing by running the `ddev config` command:

    ddev config --additional-hostnames extraname,fr.mysite,es.mysite,it.mysite,*.lotsofnames

```